src/linux/cgroups2.cpp

```diff
@@ -382,4 +404,42 @@ Try<set<string>> enabled(const string& cgroup)
 
 } // namespace controllers {
 
+namespace cpu {
+
+namespace control {
+
+const std::string IDLE = "cpu.idle";
+const std::string MAX = "cpu.max";
+const std::string MAX_BURST = "cpu.max.burst";
+const std::string PRESSURE = "cpu.pressure";
+const std::string STATS = "cpu.stat";
+const std::string UCLAMP_MAX = "cpu.uclamp.max";
+const std::string UCLAMP_MIN = "cpu.uclamp.min";
+const std::string WEIGHT = "cpu.weight";
+const std::string WEIGHT_NICE = "cpu.weight.nice";
+
+} // namespace control {
+
+Try<Nothing> weight(const string& cgroup, uint32_t weight)
+{
+  if (cgroup == ROOT_CGROUP) {
+    return Error("Operation does not exist for the root cgroup");
+  }
+
+  return cgroups2::write(cgroup, cpu::control::WEIGHT, (uint64_t) weight);
```

Comment:
this is a C style cast, we would use static_cast here to check at compile time that it works, however we probably don't need to be casting since we don't have an integer type that holds the needed range anyway